internal/project/service_test.go

```diff
@@ -384,3 +394,139 @@ func TestServiceCreateLanguagesVersion(t *testing.T) {
 		assert.ErrorIs(t, err, errTest)
 	})
 }
+
+func TestServiceGetTranslation(t *testing.T) {
+	t.Parallel()
+
+	storage := storage.NewMockStorage(gomock.NewController(t))
+	poeditClient := poedit.NewMockClient(gomock.NewController(t))
+	cacheClient := cache.NewMockCache(gomock.NewController(t))
```

Comment:
While I can see it's not done in the other tests, I would recommend instantiating the mock controller to its own variable, which you then pass to the different mocks here.

Just to ensure it doesn't have any potential to affect the test outcome if they were run with their own, separate, controllers (as they do here)